change-notes/1.24/analysis-csharp.md

```diff
@@ -27,7 +27,8 @@ The following changes in version 1.24 affect C# analysis in all applications.
 ## Changes to code extraction
 
 * Tuple expressions, for example `(int,bool)` in `default((int,bool))` are now extracted correctly.
-* Expression nullability flow state is extracted.
+* Expression nullability flow state is extracted. 
+* `stackalloc` array creations are now extracted, and they are represented by the class `Stackalloc`.
```

Comment:
This gives the impression that they weren't previously extracted at all. How about

In extraction:

* The difference between `stackalloc` array creations normal array creations is extracted.

In changes to QL libraries:

- `stackalloc` array creations are now represented by the QL class `Stackalloc`. Previously they were represented by the class `ArrayCreation`.